### PR TITLE
Fix RMSNorm scale zero-initialization

### PR DESCRIPTION
**Pull request by meemoprasad**
`RMSNorm.__init__` in `src/timesfm/flax/normalization.py` initializes
`self.scale` with `jnp.zeros` instead of `jnp.ones`:

    self.scale = nnx.Param(jnp.zeros(shape=(num_features,)))

Since `__call__` does `normed_inputs *= self.scale`, this makes
RMSNorm output all zeros on a fresh forward pass, before any
checkpoint load overwrites the parameter.

`LayerNorm` in the same file correctly initializes its scale with
`jnp.ones` — this looks like a copy-paste bug between the two classes.

Currently masked in inference when checkpoints are always loaded
before a forward pass, but affects:
- standalone instantiation of RMSNorm (e.g. in tests)
- any future from-scratch training / fine-tuning of the flax backend

Fix: `jnp.zeros` → `jnp.ones`, matching `LayerNorm`'s pattern.

### CODE REVIEW

**Comment by meemoprasad** [comment]
> Thanks for your pull request! It looks like this may be your first contribution to a Google open source project. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> View this [failed invocation](https://github.com/google-research/timesfm/pull/481/checks?check_run_id=100348683424) of the CLA check for more information.
> 
> For the most up to date status, view the checks section at the bottom of the pull request.

  CLA is now signed and verified — ready for review whenever a maintainer gets a chance.

**Review by sylvesterkaczmarek — COMMENTED** [review]
This fixes the zero-initialized scale, but there is no regression covering the failure mode. With the old initializer, a fresh `RMSNorm` maps every input to zero because the normalized activations are multiplied by an all-zero scale, and the current suite would not catch that coming back. Could we add a small test that a newly constructed layer has unit scale and preserves nonzero normalized output?
